First/ViewController.swift

```diff
@@ -15,7 +15,7 @@ class ViewController: UIViewController {
     var trouble: UIButton!
     let daymix = UIImage(named: "DaymixLogo.png")
     var logo: UIImageView!
```

Comment:
Nazwy obiektów które tworzymy powinny też w sobie zawierać rodzaj obiektu. Dzięki temu jak czyta się kod, to nie trzeba sprawdzać jego deklaracji żeby wiedzieć jakiego typu jest dana zmienna. Więc na przykład tutaj zamiast _logo_ będzie _logoImageView_.